scripts/start-servers.php

```diff
@@ -47,30 +47,39 @@ function make_ctx($preset, $method = "POST") {
     return $ctx;
 }
 
-function failed($result) {
```

Comment:
I quite liked the failed, with the print, the "last operation time" and the exit. I would rather see this repurposed than deleted.

Reply:
Restored in https://github.com/mongodb/mongo-php-driver/pull/856/commits/eafc29a054a01fc167112fbbd34fd8f42d0d7083.